initializers/users.yml

```diff
@@ -4,6 +4,7 @@
 #   password: reader
 # writer:
 #   password: writer
+#   api_token:  # leave empty to not generate any token
```

Comment:
Maybe this could be made mode explicit:

```suggestion
#   api_token: "" # a token is generated automatically unless the value is explicity set to empty
```